code/script.js

```diff
@@ -222,86 +223,148 @@ const generateBoard = () => {
   })
 }
 
+
 // Randomly select a person from the characters array and set as the value of the variable called secret
 const setSecret = () => {
   secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
+  console.log('secret person', secret)
 }
 
+
 // This function to start (and restart) the game
 const start = () => {
   // Here we're setting charactersInPlay array to be all the characters to start with
   charactersInPlay = CHARACTERS
-  // What else should happen when we start the game?
+  // What else should happen when we start the game? 
 }
 
-// setting the currentQuestion object when you select something in the dropdown
-const selectQuestion = () => {
-  const category = questions.options[questions.selectedIndex].parentNode.label
 
+  // setting the currentQuestion object when you select something in the dropdown
   // This variable stores what option group (category) the question belongs to.
   // We also need a variable that stores the actual value of the question we've selected.
-  // const value =
-
-  currentQuestion = {
-    category: category,
-    // value: value
-  }
-}
+const selectQuestion = () => {
+  const category = questions.options[questions.selectedIndex].parentNode.label
+  const value = questions.value
+  console.log('this is a test for category and value')
 
-// This function should be invoked when you click on 'Find Out' button.
-const checkQuestion = () => {
-  const { category, value } = currentQuestion
+    if (category === 'hair') {
+      console.log('hair', value)
+      currentQuestion = {
+        category: category,
+        value: value
+        }
+      }
+   else if (category === 'eyes') {
+        console.log('eyes', value)
+        currentQuestion = {
+          category: category,
+          value: value
+        }
+      }
+    else if (category === 'accessories') {
+        console.log('accessories', value)
+        currentQuestion = {
+          category: category,
+          value: value
+        }
+      }
+    else if (category === 'other') {
+        console.log('other', value)
+        currentQuestion = {
+          category: category,
+          value: value
+        }  
+      }
+    }
+    
+selectQuestion()
```

Comment:
I think you might have hade a slight head-spin about this and made it more complicated than it needs to be. At the moment the browser doesn't really know what to do and therefor does nothing. 
This is how I solved it in my code:
````
   // The function that sets the currentQuestion object when you select something in the dropdown
    const selectQuestion = () => {
        const category = questions.options[questions.selectedIndex].parentNode.label;
        const value = questions.options[questions.selectedIndex].value;

        currentQuestion = {
            category: category,
            value: value
        }
        checkQuestion();    // Invokes the checkQuestion-function 
        }
````